.kitchen.yml

```diff
@@ -23,4 +23,12 @@ suites:
         - name: chef-aps-inspec-tomcat
           git: https://github.com/Alfresco/chef-aps-inspec-tomcat
         - path: test/inspec/
-    attributes:
+    attributes: {
```

Comment:
can we have it in a different suite? ssl enabled should not be in the default suite